.github/workflows/single-binary.yml

```diff
@@ -175,19 +175,38 @@ jobs:
               memory: "0"
           EOF
           flytectl demo start --image flyte-sandbox-bundled:local --imagePullPolicy Never
-      - name: Register examples
-        uses: unionai/flyte-register-action@v0.0.2
-        with:
-          flytesnacks: true
-          config: boilerplate/flyte/end2end/functional-test-config.yaml
-          project: flytesnacks
-          domain: development
-          version: ${{ env.FLYTESNACKS_VERSION }}
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
           pip install flytekit
           pip freeze
+      - name: Checkout flytesnacks
+        uses: actions/checkout@v3
+        with:
+          repository: flyteorg/flytesnacks
+          path: flytesnacks
```

Comment:
Should we `FLYTESNACKS_VERSION` in this checkout step? We can update the variable with the latest commit sha if the original value was a branch like `master`.

Reply:
yup, left a comment in the next update explaining this.